docs/index.md

```diff
@@ -41,10 +54,47 @@ $ docker-machine -v
 machine version 0.1.0
 ```
 
+In order to run Docker commands on your machines without having to use SSH, make
+sure to install the Docker client as well, e.g.:
+
+```
+$ curl https://get.docker.com/builds/Darwin/x86_64/docker-latest > /usr/local/bin/docker
+```
+
```

Comment:
Eliminate the passive: "Currently, Docker recommends that you install Machine on Windows using mysysgit..."